hostinit/setup.go

```diff
@@ -343,9 +343,8 @@ func (init *HostInit) setupHost(ctx context.Context, targetHost *host.Host) (str
 	if err = targetHost.SetInitializing(); err != nil {
 		if err == mgo.ErrNotFound {
 			return "", ErrHostAlreadyInitializing
-		} else {
-			return "", errors.Wrapf(err, "database error")
```

Comment:
This should return _something_ if there's an error.

Reply:
we do on line 347,